packages/figma-plugins/common/package.json

```diff
@@ -9,15 +9,18 @@
     "test": "echo 'Tests are not implemented yet'",
     "tsc": "tsc --noEmit -p src/main",
     "tsc:watch": "yarn tsc:main -- --watch --preserveWatchOutput",
-    "build": "rimraf ./dist && esbuild src/* --outdir=dist"
+    "build": "rimraf ./dist && yarn build:ui && yarn build:main",
+    "build:main": "esbuild src/main/* --outdir=dist/main",
+    "build:ui": "esbuild src/ui/* --outdir=dist/ui"
   },
   "dependencies": {
     "buffer": "^6.0.3",
+    "p-queue": "^6.6.2",
```

Comment:
That's not needed since we have `Queue` in `octopus-common` which is much more lightweight than `p-queue` (which uses EE3 + `p-timeout` as deps). Please rewrite with `Queue`.

Reply:
Figma app is missing `queueMicrotask`, so we probably cant use our Queue now.